lua/starfall/libs_sh/mesh.lua

```diff
@@ -762,6 +775,199 @@ if CLIENT then
 		end
 	end
 
+	--- Starts a new dynamic mesh. If an Mesh object is passed, it will use that mesh instead
+	-- @param mesh Optional Mesh object, mesh to build, can be nil to build a 'dynamic' mesh
+	-- @param prim_type Int, primitive type, see MATERIAL
+	-- @param prim_count Int, the amount of primitives
+	-- @client
+	function mesh_library.begin(mesh, prim_type, prim_count)
+		local data = instance.data
```

Comment:
instance.data.mesh_dynamic and others can be a local defined above these functions

Reply:
should probably keep refrence on instance and localise it too, that way other addons can have access to it

Reply:
That seems unnecessary.